src/bulletin/bulletin.service.ts

```diff
@@ -1,7 +1,8 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, NotFoundException } from '@nestjs/common';
 import { PrismaService } from '../prisma/prisma.service';
 import { SupabaseService } from '../../supabase/supabase.service';
 import { AddBulletinDTO } from './createBulletin.dto';
+import { da } from '@faker-js/faker';
```

Comment:
Unused import? @SaaammmyyyS  pa add ko sa prettier or linter to remove unused imports